pom.xml

```diff
@@ -97,18 +97,6 @@
 			<version>3.2.0</version>
 		</dependency>
 
-		<dependency>
```

Comment:
AFAIK these dependencies are being used, right? Generally, I think we should declare dependencies explicitly and not rely on transitivity if the corresponding project uses them directly.